Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -76,10 +83,28 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
     
     public func isEnabled(featureKey: PrivacyFeature,
                           versionProvider: AppVersionProvider = AppVersionProvider()) -> Bool {
-        guard let feature = data.features[featureKey.rawValue] else { return false }
-        
-        return satisfiesMinVersion(feature: feature, versionProvider: versionProvider)
-                && feature.state == PrivacyConfigurationData.State.enabled
+        guard let feature = data.features[featureKey.key] else { return false }
+        let satisfiesMinVersion = satisfiesMinVersion(feature.minSupportedVersion, versionProvider: versionProvider)
+        switch feature.state {
+        case PrivacyConfigurationData.State.enabled: return satisfiesMinVersion
+        case PrivacyConfigurationData.State.internal: return internalUserDecider.isInternalUser && satisfiesMinVersion
+        default: return false
+        }
+    }
+
+    public func isSubfeatureEnabled<F: NestedFeature, SF>(for feature: F.Type, _ subfeature: SF, versionProvider: AppVersionProvider) -> Bool where F.SubFeatureType == SF {
```

Comment:
If the parent feature is set to disabled, but the subfeature is set to enabled, should the sub feature resolve to enabled?
I think it probably shouldn't be, did we get alignment on that?

Reply:
You're right, good catch. This was an oversight but it's a simple a fix as just guarding the parent feature's value first.